backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/TaskServiceInDatabase.java

```diff
@@ -0,0 +1,265 @@
+package org.molgenis.emx2.tasks;
+
+import static org.molgenis.emx2.Column.column;
+import static org.molgenis.emx2.FilterBean.f;
+import static org.molgenis.emx2.FilterBean.or;
+import static org.molgenis.emx2.Operator.EQUALS;
+import static org.molgenis.emx2.Row.row;
+import static org.molgenis.emx2.TableMetadata.table;
+
+import java.io.File;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.molgenis.emx2.*;
+import org.molgenis.emx2.sql.SqlDatabase;
+
+public class TaskServiceInDatabase extends TaskServiceInMemory {
+  private Schema systemSchema;
+
+  public TaskServiceInDatabase(Schema systemSchema) {
+    // for testing make parameterizable
+    this.systemSchema = systemSchema;
+    this.init();
+  }
+
+  public TaskServiceInDatabase() {
+    // default uses ADMIN schema and dedicated database instance
+    Database database = new SqlDatabase(false);
+    database.becomeAdmin();
+    if (!database.hasSchema("ADMIN")) {
+      // assumes database is never changed by humans
```

Comment:
the 'default' database you mean?

Reply:
to be discussed, but implicit in this PR is to create a special 'ADMIN' schema for database tables supporting platform logic beyond the metadata tables.